types/passport-google-oauth20/tsconfig.json

```diff
@@ -4,10 +4,8 @@
         "lib": [
             "es6"
         ],
-        "noImplicitAny": true,
-        "noImplicitThis": true,
-        "strictNullChecks": true,
-        "strictFunctionTypes": true,
```

Comment:
Usually people add stuff to the tsconfig to make things less strict, which is why the bot marks it as something to manually check.  In this case, this is adding a *stricter* setting, which [the bot](https://github.com/DefinitelyTyped/dt-mergebot/) should arguably ignore.  It will be useful if you add a quick issue there about it so we can extend it at some point...

(This is just a technicality point, since it means that this change will always make the bot wait for a maintainer for no good reason.)